packages/vue2/src/init.js

```
@@ -63,11 +63,19 @@ const init = (appConfig, platformConfig, routes, metaData) => {
     link.href = platformConfig?.documentIcon;
   }
 
-  // 应用初始化之前 不能访问应用中的任何逻辑
-  evalWrap.bind(window)(metaData, "rendered");
-  ["preRequest", "postRequest"].forEach((fnName) => {
-    evalWrap.bind(window)(metaData, fnName);
-  });
+  // 注册端事件
+  const endEventLists = ["rendered", "beforeRouter", "afterRouter", "preRequest", "postRequest"];
+  if (metaData && metaData.frontendEvents) {
+    for (let index = 0; index < endEventLists.length; index++) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)